_adr/83/index.adoc

```diff
@@ -0,0 +1,158 @@
+---
+num: 83
+title: Can-I API
+status: "Accepted"
+authors:
+  - "Massimiliano Ziccardi"
+tags:
+  - "kafka"
+#applies_padrs: # What PADRs does this ADR apply?
+#applies_patterns: # What APs does this ADR apply?
+---
+## Context and Problem Statement
+Deciding whether a certain operation is allowed can be a complex matter and may require interactions with different components.
+
+This new API will allow the clients to check whether a certain operation is allowed by simply passing the top-level verb (create,delete,list, etc) and receiving whether it is allowed and what are the granted resources.
```

Comment:
```suggestion
The clients need to be able to check whether a certain operation is allowed without having all or any of the details about the instance the user wants to create (because the user hasn't completed the form yet).
So the only available inputs are the top-level verb (create, delete, list, etc.) and the resource (e.g. kafka instance)
```